indico/modules/events/templates/emails/event_creation.txt

```diff
@@ -0,0 +1,57 @@
+{% extends 'emails/base.txt' %}
+
+{% block subject -%}
+    New {{ event.type }} in Indico
+{%- endblock %}
+
+{% block header -%}{%- endblock %}
+
+{% block body -%}
+_Title_
+    {{ event.title }}
+
+_Category_
+    {{ event.category.chain_titles|join(' » ') }}
+
+{% if event.person_links -%}
+{{ '_Speakers_' if event.type == 'lecture' else '_Chairpersons_' }}
+    {{ event.person_links|sort(attribute='full_name')|join('; ', attribute='full_name') }}
+{%- endif %}
+
+{% if event.room_name -%}
+    _Room_
+    {{ event.room_name }}
+{%- endif %}
+
```

Comment:
There's still lots of awful whitespace if some of these items aren't missing. You need to add the linebreak within the if block, not outside